mkenv/tests/test_find.py

```diff
@@ -24,8 +24,9 @@ def test_find_directory_defaults_to_cwd(self):
         self.assertEqual(
             (stdin, stdout, stderr), (
                 "",
-                str(self.locator.for_directory(Path.from_string(".")).path) +
-                "\n",
+                str(self.locator.for_directory(
```

Comment:
This change confuses me a bit, but I think you're bumping into a bug (one that isn't here).

Basically if you see "RelativePath has no attribute basename" that's indicating that Filesystems is mis-parsing an absolute path (which should return objects of type `Path`) as a relative one (which uses `RelativePath`). The latter is missing some methods of the former (like basename), though in theory it should have all of them, but I'm not adding them yet until we fix Windows support because seeing that traceback is a good (albeit slightly obscure) way of telling that an absolute path was mis-parsed.

Lemme know if that makes sense -- is `Path.from_string(".")` the Windows path that's returning a `RelativePath`?

Reply:
Hmm so actually, I think the real solution here is to just switch to `Path.cwd`, which should return the absolut-ized path of cwd (using the same mechanism you have here).

I guess yeah `Path.from_string(".")` should return a relative one for better or worse.